src/addons/dragAndDrop/EventContainerWrapper.js

```diff
@@ -271,8 +246,10 @@ class EventContainerWrapper extends React.Component {
       ),
     })
   }
-}
 
-EventContainerWrapper.propTypes = propTypes
+  render() {
+    return <div ref={this.ref}>{this.renderContent()}</div>
```

Comment:
i don't think spliting renderContent out as a separate function adds much? maybe just have `render()`.

On the another note, I think it might more more sense to pass the ref to `children` instead of adding a new DOM element

Reply:
Absolutely right. I was having trouble forwarding the ref and took the cowards way out. I'll try to work on this.